LICENSE.md

```diff
@@ -1,6 +1,11 @@
 The Oscar.jl package is licensed under the GNU Public License, Version 3.0+:
 
-> Copyright (c) 2019 The OSCAR Development Team.
+> Copyright (c) 2019-2024 The OSCAR Development Team
+>
+> See <https://www.oscar-system.org/contributors/> for a list of OSCAR
+> Development Team members. Note that this list may occasionally be slightly
+> outdated. Additional contributors may be found in the `git log` of the main
+> OSCAR repository and other related repositories.
```

Comment:
Also relevant: https://github.com/oscar-system/Oscar.jl/issues/277 -- perhaps that can be closed with this PR (and with the work by @HereAround and @aaruni96 on https://github.com/oscar-system/oscar-website/pull/373)